scripts/globals/items/scroll_of_suiton_san.lua

```diff
@@ -0,0 +1,13 @@
+-----------------------------------------
+-- ID: 4945
+-- Scroll of Suiton: Ni
+-- Teaches the ninjutsu Suiton: Ni
```

Comment:
Update to San (verify the spell numbers are correct?)

Reply:
Still waiting on this one, but everything else looks good! :+1: